src/custom/pages/Faq/index.tsx

```diff
@@ -408,16 +408,38 @@ export default function Faq() {
             orderbook and place counter orders (creating a CoW) to prevent settling trades via external liquidity.
           </p>
           <h3 id="wallet-not-supported">Why is my wallet not supported?</h3>
-          <p>CowSwap uses offline signatures to offer gasless orders.</p>
           <p>
-            Currently, Smart Contract (SC) wallets such as Gnosis Safe, Argent or Pillar are not supported because it
-            would require signing an on-chain transaction to place the order, making it no longer gasless. We are
-            working to make this a possibility and support will be added soon.
+            CowSwap uses offchain signatures to offer gasless orders, additionally has support for smart contract
+            wallets by using some alternative signing method called{' '}
+            <LinkScrollable href={'#what-is-presign'}>pre-sign</LinkScrollable>.
           </p>
           <p>
-            Nevertheless, even if your wallet is not an SC wallet, it might be unsupported in some cases. Not all
-            wallets implement the necessary signing methods from EIP712 standard. If that is the case for you, reach out
-            to your wallet developers and ask for it.
+            Smart Contract (SC) are supported by using pre-sign method, however for normal wallets (EOA) CowSwap
+            requires the wallet to support offchain-signing (EIP712 standard).
+          </p>
+          <p>
+            Some wallets have reported issues with this offchain-signing, so they disabled it. If that is the case for
+            your wallet, reach out to your wallet provider and ask for it to be implemented.
+          </p>
+          <h3 id="smart-contract-support">Are smart contract integrations supported?</h3>
+          <p>Yes! Any smart contract can trade in CowSwap by using the signing method called pre-sign.</p>
+          <p>
+            pre-sign is a protocol operation that can be invoked by any contract. The operation has a single parameter
+            that is the &quot;orderId&quot; which identifies the order that the smart contract is approving. Once the
+            smart contract pre-signs an order, it becomes automatically tradable, therefore solvers will start
+            considering the order for their settlement solutions.
+          </p>
+          <p>
+            In the future, the protocol could provide EIP712 support for off-chain signing also for smart contracts,
```

Comment:
I believe this was mentioned elsewhere. It's not EIP712 for SC. Is it https://eips.ethereum.org/EIPS/eip-2771 maybe?